lib/fog/google/requests/storage_json/put_object_acl.rb

```diff
@@ -0,0 +1,49 @@
+module Fog
+  module Google
+    class StorageJSON
+      class Real
+        # TODO: move this methods to helper to use them with put_bucket_acl request
+        def tag(name, value)
+          # "<#{name}>#{value}</#{name}>"
+        end
+
+        def scope_tag(scope)
+          # if %w(AllUsers AllAuthenticatedUsers).include?(scope["type"])
+          #   "<Scope type='#{scope['type']}'/>"
+          # else
+          #   "<Scope type='#{scope['type']}'>" +
+          #     scope.to_a.select { |pair| pair[0] != "type" }.map { |pair| tag(pair[0], pair[1]) }.join("\n") +
+          #     "</Scope>"
+          # end
+        end
+
+        def entries_list(access_control_list)
+          # access_control_list.map do |entry|
+          #   tag("Entry", scope_tag(entry["Scope"]) + tag("Permission", entry["Permission"]))
+          # end.join("\n")
+        end
+
+        def put_object_acl(bucket_name, object_name, acl)
```

Comment:
Unused method argument - `bucket_name`. If it's necessary, use `_` or `_bucket_name` as an argument name to indicate that it won't be used. You can also write as `put_object_acl(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `object_name`. If it's necessary, use `_` or `_object_name` as an argument name to indicate that it won't be used. You can also write as `put_object_acl(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `acl`. If it's necessary, use `_` or `_acl` as an argument name to indicate that it won't be used. You can also write as `put_object_acl(*)` if you want the method to accept any arguments but don't care about them.